ansible_ai_connect/healthcheck/backends.py

```diff
@@ -141,32 +113,37 @@ def identifier(self):
         return self.__class__.__name__
 
 
-class ChatbotServiceHealthCheck(BaseLightspeedHealthCheck):
```

Comment:
The ChatBot health check has [moved](https://github.com/ansible/ansible-ai-connect-service/pull/1431/commits/4628048859cb69e1952b978b10f33cfeb2d20c73#diff-b6c900f30e6213219c6ddcd3af4d9405581fee6029a05e493bdd4d9613268578R171-R198) to it's _pipeline_.